sda-commons-starter-web/src/main/java/org/sdase/commons/spring/boot/web/monitoring/SdaTracingConfiguration.java

```diff
@@ -38,7 +40,14 @@ public TextMapPropagator jaegerPropagator() {
 
   @Bean
   @ConditionalOnProperty(value = "management.tracing.grpc.enabled", havingValue = "true")
-  public OtlpGrpcSpanExporter otlpGrpcExporter() {
-    return OtlpGrpcSpanExporter.getDefault();
+  public OtlpGrpcSpanExporter otlpGrpcExporter(
+      @Value("${management.tracing.grpc.endpoint}") String endpoint,
+      @Value("${management.tracing.grpc.compression}") String compression,
+      @Value("${management.tracing.grpc.timeoutMs}") Long timeout) {
```

Comment:
Do we define these properties or is there a configuration class or at least some constants we could use?

Reply:
Sorry, I forgot to create as a Draft. Now it is ready for review